src/SDKs/CognitiveServices/dataPlane/Search/BingEntitySearch/BingEntitySearch/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Entity Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Entity Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
Please do not update the `AssemblyVersion` here

Reply:
Undone